test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -349,7 +356,11 @@ def create_pytorch_only_extra_kwargs():
             return {}
 
         self._test_large_scale_exporter(
-            "toy_mlp1", create_model, create_args, create_pytorch_only_extra_kwargs
+            "toy_mlp1",
+            create_model,
+            create_args,
+            create_pytorch_only_extra_kwargs,
+            dynamic_axes=True,
```

Comment:
Let's test both of `dynamic_axes=True` and `dynamic_axes=False`.

Reply:
I thought symbolic mode is the top choice of exporter? I see dynamic_axes=False(fake mode) as a back plan. In other words, why do we need to main fake mode if we know symbolic mode is working?

Reply:
If you don't test backup plan, how do you know backup plan will work?

Reply:
Well, I was thinking we don't need other modes anymore if we have symbolic. But in these two large scale cases, they need to export with FakeTensorMode, which it seems to disallow us use symbolic in later dynamo/make_fx. So either way, this should be False.